Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift

```diff
@@ -181,6 +183,25 @@ extension BrowserViewController {
     present(controller, animated: true)
   }
   
+  private func presentCookieNotificationBlockingCallout(skipSafeGuards: Bool = false) {
```

Comment:
Do we show this popup for users who already saw the original 'opt in' popup?

Reply:
Again, no. It used the same setting as before as the name fit